pageserver/src/walredo.rs

```diff
@@ -120,7 +120,7 @@ fn can_apply_in_neon(rec: &NeonWalRecord) -> bool {
 /// An error happened in WAL redo
 #[derive(Debug, thiserror::Error)]
 pub enum WalRedoError {
-    #[error(transparent)]
+    #[error("encountered io error: {0}")]
```

Comment:
This seems unrelated

Reply:
That was helpful during local debugging, I can extract it to separate PR. With transparent it will just yield the io Error without mentioning WalRedoError at all